src/components/Calculator.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+// eslint-disable-next-line react/prefer-stateless-function
+class Calculator extends React.Component {
+  render() {
+    return (
+      <div className="calculator">
+        <div className="calc-output"> 0 </div>
+        <div className="calc-grid">
+          <button type="button"> AC </button>
+          <button type="button"> +/- </button>
+          <button type="button"> % </button>
+          <button type="button" className="logic-operators"> ÷ </button>
+          <button type="button"> 7 </button>
+          <button type="button"> 8 </button>
+          <button type="button"> 9 </button>
+          <button type="button" className="logic-operators"> x </button>
+          <button type="button"> 4 </button>
```

Comment:
- [ ] The `calculator output` is a bit broken, please update your styling so it matches the wireframe.